azure_deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 echo "Deploying app to Azure";
-curl -dH -X POST $AZURE_APP_WEBHOOK_URL
\ No newline at end of file
+curl -dH -X POST $deploy_webhook_url
```

Comment:
I don't think this variable is actually set anywhere is it? I think you want to use:
```suggestion
curl -dH -X POST "$(terraform output -raw cd_webhook)"
```

Reply:
done